src/data_clusterer/algorithms.py

```diff
@@ -0,0 +1,54 @@
+from typing import Any
+
+import numpy.typing as npt
+from sklearn.cluster import KMeans
+
+
+class KMeansAlgorithm:
+    """Class for setting up and running KMeans clustering algorithm.
+
+    Attributes:
+        model (KMeans): An instance of a KMeans object.
+
+    """
+
+    def __init__(self, **kwargs) -> None:
+        """Initialization of the model and hyperparameters' attributes.
+
+        First model parameter is initialized with KMeans object. Then,
+        non-None values passed as keywords arguments are assigned to the
+        corresponding KMeans class attributes. If any attribute is not set,
+        default settings of KMeans class is used for the attribute.
+
+        Args:
+            **kwargs (dict): Keyword arguments containing name - value pairs
```

Comment:
please select some parameters  and use them explicitly as parameters - explicit is better than implicit